src/components/Activities/Activity3/Game/index.tsx

```diff
@@ -38,6 +47,16 @@ function Game(): JSX.Element {
     const tempVariables = storedVariables?.split(',').map(element => element as VARIABLES);
     const curVariables = tempVariables ?? [];
     setVariableSelection(curVariables);
+
+    // setup tasksSelection from storage
+    const tempTasks : number[] = storedTasks ?
+      storedTasks.split(',').map((elem) => {
+        // elem can be either '' or a number
+        return (elem && elem.length > 0) ? parseInt(elem) : 0;
+      })
+      : [0,0,0];
```

Comment:
`object?.command()` --> object runs command if object is not null!
`object ?? [value]` --> returns `[value]` if object is null  
```suggestion
    const tempTasks : number[] = storedTasks?.split(',').map((elem) => {
        // elem can be either '' or a number
        return (elem && elem.length > 0) ? parseInt(elem) : 0;
      })
      ?? [0,0,0];
```

Reply:
oh good to know!